tests/migrator/test_books.py

```diff
@@ -2825,3 +2825,21 @@ def test_conference_info_multiple_series_number(app):
                 """,
                 dict(),
             )
+
+
+def test_open_access(app):
```

Comment:
it would be great to add tests for all cases: `open_access: True/False`, `login_required: True/False`

Reply:
login required is added on the import time, depending on the link. There is no tests yet for that, it would be a bigger work, not for this PR